src/app/boot/app_controller.nim

```diff
@@ -467,7 +466,6 @@ proc load(self: AppController) =
     self.chatService,
     self.communityService,
     self.messageService,
-    self.gifService,
```

Comment:
Hi there, thanks for the changes, do you mind explaining what is the new architecture design? What are some triggers and consequences of this change please? Does that mean for instance that the main module will be slower to start?